src/main/scala/outwatch/dom/helpers/TargetOps.scala

```diff
@@ -0,0 +1,27 @@
+package outwatch.dom.helpers
+
+import org.scalajs.dom.{Event, EventTarget, html}
+
+trait TargetOps {
+
+  implicit class EventTargetAsInput[E <: Event, ET <: EventTarget](builder: EmitterBuilder[E, ET]) {
+    def value: EmitterBuilder[E, String] = builder.map(_.asInstanceOf[html.Input].value)
+
+    def valueAsNumber: EmitterBuilder[E, Int] = builder.map(_.asInstanceOf[html.Input].valueAsNumber)
+
+    def checked: EmitterBuilder[E, Boolean] = builder.map(_.asInstanceOf[html.Input].checked)
+  }
+
+  implicit class CurrentTargetAsInput[E <: Event, O <: Event](builder: EmitterBuilder[E, O]) {
+    def value: EmitterBuilder[E, String] = builder.map(e => e.currentTarget.asInstanceOf[html.Input].value)
+
+    def valueAsNumber: EmitterBuilder[E, Int] = builder.map(e => e.currentTarget.asInstanceOf[html.Input].valueAsNumber)
+
+    def checked: EmitterBuilder[E, Boolean] = builder.map(e => e.currentTarget.asInstanceOf[html.Input].checked)
+  }
+
+  implicit class TargetMap[E <: Event, O <: Event](builder: EmitterBuilder[E, O]) {
```

Comment:
I would argue against mapping to EventTarget here, as `_.map(_.target)` seems good enough to me and we do not have the same helper functions for other event properties. Instead we can only define a target object with the above methods in an implicit class for the EmitterBuilder.

Like this:
```scala
  implicit class TargetAsInput[E <: Event, O <: Event](builder: EmitterBuilder[E, O]) {
      object target {
          def value = ???
          def valueAsNumber = ???
          def checked = ???
      }
￼   }
```

Reply:
Good point, I'll update the PR.